owners/test/github.test.js

```diff
@@ -180,7 +180,10 @@ describe('GitHub API', () => {
 
   describe('customRequest', () => {
     beforeEach(() => {
-      sandbox.stub(process, 'env').value({GITHUB_ACCESS_TOKEN: '_TOKEN_'});
+      sandbox.stub(process, 'env').value({
+        ...process.env,
```

Comment:
Would it not work to set `NODE_ENV` to `test` here instead of using the spread operator?

Reply:
It would work equally well. It seemed reasonable to inherit env vars (like LOG_LEVEL), and then the individual stubs wouldn't have to know about stubbing that particular env var (it's handled by probot, and here we're just overwriting process.env). So, this is taking the existing process.env and stubbing the particular property we want, rather than the whole environment. I could go either way on which is preferable.

Reply:
My preference is to stub just the `NODE_ENV` variable in this PR. This way, it will be an explicit decision to stub another variable in future.

Reply:
Done.